docs/source/preferences/Preferences.rst

```diff
@@ -245,14 +229,14 @@ preferences is just like using the plain old non-scoped version::
   >>> from apptools.preferences.api import ScopedPreferences
   >>> preferences = ScopedPreferences(filename='example.ini')
   >>> preferences.load('example.ini')
-  >>> p.dump()
+  >>> preferences.dump()
 
-    Node() {}
-      Node(application) {}
-        Node(acme) {}
-          Node(ui) {'bgcolor': 'blue', 'ratio': '1.0', 'width': '50', 'visible': 'True'}
-            Node(splash_screen) {'image': 'splash', 'fgcolor': 'red'}
-      Node(default) {}
+   Node() {}
+     Node(application) {}
+       Node(acme) {}
+         Node(ui) {'bgcolor': 'blue', 'width': '50', 'ratio': '1.0', 'visible': 'True'}
```

Comment:
Changed because since Python 3.7 dictionary order is preserved to match insertion order. So the output order is going to match the order in the preference file.